### PR TITLE
Compat BWA3

### PR DESCRIPTION
**Pull request by ulteq**
Compatibility PBO for BW-Mod weapon, ammo and optic classes.

### CODE REVIEW

**Comment by ulteq** [comment]
@commy2, @KoffeinFlummi please don't kill me for adding this and let me know if you want something changed.

**Comment by esteldunedain** [comment]
> @commy2, @KoffeinFlummi please don't kill me for adding this and let me know if you want something changed.

Nah, don't worry. We used to mess with BWMod too on the AGM Compatibility project.

**Comment by KoffeinFlummi** [comment]
I won't kill you for adding this but I will kill you for the tabs :stuck_out_tongue:

File: optionals/compat_bwa3/CfgAmmo.hpp [hunk 1]
```diff
@@ -0,0 +1,148 @@
+class CfgAmmo
+{
```
Comment (bux578) [inline optionals/compat_bwa3/CfgAmmo.hpp]: parantheses not on a new line

**Comment by Raspu86** [comment]
Stats for BWA3_B_762x51_Ball_LR are switched with BWA3_B_762x51_Ball_AP. It should be the other way around.

**Pull request merged**